src/components/standalone/monitoring/TrafficMonitor.vue

```diff
@@ -156,7 +153,15 @@ const hoursDatasets = computed(() => {
             <SimpleStat> {{ get('protocol').toUpperCase() }}</SimpleStat>
           </NeCard>
           <NeCard v-if="contains('host')" :title="t('standalone.real_time_monitor.remote_hosts')">
-            <SimpleStat> {{ get('host') }}</SimpleStat>
+            <SimpleStat>
+              <p class="text-lg">{{ get('host') }}</p>
+            </SimpleStat>
+          </NeCard>
+          <NeCard :title="t('standalone.real_time_monitor.daily_total_traffic')">
+            <SimpleStat>
+              <span v-if="data.total_traffic != 0">{{ byteFormat1024(data.total_traffic) }}</span>
+              <span v-else> N/A </span>
```

Comment:
In this cases we usually use "-". If you prefer we can show an internationalized "Not available", but I would avoid a static "N/A"